test/WebJobs.Script.Tests/Extensions/ConfigurationExtensionsTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using Microsoft.Azure.WebJobs.Script.Extensions;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Extensions
```

Comment:
Can you add a test showing the service bus and event hub payload that is being sent? I remember seeing this in a previous commit, perhaps I missed it? Mainly just to verify that `maxConcurrentCalls` and `maxBatchSize` are sent correctly for service bus and event hub, respectively.

Reply:
Following up on these two specifically because we need these values for target based scaling.
```
{
  "extensions": {
    "serviceBus": {
      "maxConcurrentCalls": 100
    },
    "eventHubs": {
      "eventProcessorOptions": {
        "maxBatchSize": 200
      }
    }
  }
}
```

Reply:
@chiangvincent  found that root cause ServiceBus doesn't work was, they have different structure for V5+ and old versions.